converters/cfloop.cfc

```diff
@@ -47,6 +47,18 @@ component extends="BaseConverter" {
 				s = s & "++";
 			}
 			s = s & " ) {";
+		} else if (structKeyExists(attr, "query")) {
+			s = "cfloop( query= " & attr.query;
```

Comment:
If you extend `BaseBlockTagConverter` for this component instead, I think you can just use `super.toScript(tag)` here.